majortom_gateway/gateway_api.py

```diff
@@ -63,33 +65,15 @@ def __build_endpoints(self):
                 ssl_context.check_hostname = False
                 ssl_context.verify_mode = ssl.CERT_NONE
 
-        else:
-            ssl_context = None
-
         logger.info("Connecting to Major Tom")
-        try:
```

Comment:
Where did this logic go?

Reply:
It went into the retry function where connect gets called. It was already there, but I forgot to remove it from the connect function

Reply:
https://github.com/kubos/majortom_gateway_package/pull/4/files#diff-c4df29f84c49bdeab6779aff2a0bd400L103